README.md

```diff
@@ -292,8 +292,30 @@ The CPU has many specialized registers, which we don't access directly. One of t
 ## The Math Section
 If you thought you'd get through this without doing any math, well, I'm sorry. We have to do a little bit so that we can understand what the computer is doing, because like I said, it's all just basic math underneath. Now, I promise you it won't be too hard. You may get a little confused and your brain may hurt, but just stick with me here and we'll make it through to the assembly section.
 
-### Binary
+### Number Systems
+<p align="center">
+  <br>
+  <img width="460" height="300" src="https://cloud-ebbg83jzl-hack-club-bot.vercel.app/0diagram.png">
```

Comment:
I love the addition of the number systems section! We will want to fill this all out a bit more, but a diagram is a great idea.

I find this diagram to be a little spooky for some reason - maybe it's because I'm a little afraid of math. I think it's a good placeholder for now, but I bet you and I can make a more approachable one later.

Reply:
haha maybe cuz it has no colors and no explanations haha... just demonstrations. i think smth like this is also super cool! source: [math is fun](https://www.mathsisfun.com/binary-number-system.html)
<img width="681" alt="Screenshot 2022-05-26 at 10 48 53 PM" src="https://user-images.githubusercontent.com/65808924/170513124-de4f13cd-86d4-4456-a173-a3aa64804fd0.png">

Reply:
Omg I loooove how this site is explaining binary. We should totally use this as an example!!